src/schematools/types.py

```diff
@@ -99,38 +112,38 @@ def url_prefix(self) -> str:
         return self.get("url_prefix", "")
 
     @property
-    def identifier(self):
+    def identifier(self) -> str:
         """Which fields acts as identifier. (default is Django "pk" field)"""
         return self.get("identifier", "pk")
 
     @property
-    def version(self):
+    def version(self) -> str:
         """Dataset Schema Version"""
         return self.get("version", None)
 
     @property
-    def default_version(self):
+    def default_version(self) -> str:
         """Default version for this schema"""
         return self.get("default_version", self.version)
 
     @property
-    def is_default_version(self):
+    def is_default_version(self) -> bool:
         """Is this Default Dataset version.
         Defaults to True, in order to stay backwards compatible."""
         return self.default_version == self.version
 
     @property
-    def auth(self):
+    def auth(self) -> Optional[Union[str, List[str]]]:
```

Comment:
laat maar weg, ik heb hier consistent `set[str]` van gemaakt in de auth refactor.